cli/common/envoy/http.go

```diff
@@ -69,6 +73,53 @@ type Secret struct {
 	LastUpdated string
 }
 
+// CallLoggingEndpoint requests the logging endpoint from Envoy Admin Interface for a given port
```

Comment:
Can you add some description of the fact that we both retrieve and update loggers with this call?

Reply:
can do